packages/next/package.json

```diff
@@ -82,10 +82,12 @@
     "etag": "1.8.1",
     "find-cache-dir": "3.3.1",
     "jest-worker": "24.9.0",
+    "klona": "^2.0.4",
```

Comment:
Seems like this was added by mistake before we had subasset bundling? This is a dep of `sass-loader`, which should be fully compiled.

Reply:
No, this is a TODO for tree optimization - to ensure maximum compat I only inlined these packages to the first level, thus their dependencies all become direct dependencies.